Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -622,6 +623,17 @@ def test_create_subprocess_shell_text_mode_fails(self):
         self.loop.run_until_complete(execute())
 
 
+    def test_create_subprocess_exec_with_path(self):
+        async def execute():
+            p = await subprocess.create_subprocess_exec(
+                pathlib.Path(sys.executable), '-c', 'pass')
```

Comment:
If we test for the support of the path protocol it is better to use `test.support.FakePath` instead of `pathlib.Path`.